curriculum/challenges/english/09-information-security/information-security-projects/anonymous-message-board.md

```diff
@@ -122,7 +122,23 @@ You can send a POST request to `/api/replies/{board}` with form data including `
 You can send a GET request to `/api/threads/{board}`. Returned will be an array of the most recent 10 bumped threads on the board with only the most recent 3 replies for each. The `reported` and `delete_password` fields will not be sent to the client.
 
 ```js
-
+async (getUserInput) => {
+  const url = getUserInput('url');
+  const checkData = await fetch(url + '/api/threads/fcc_test');
+  const parsed = await checkData.json();
+  if(!Array.isArray(parsed)) throw new Error(`An array needs to be sent to the client got ${typeof parsed}.`);
```

Comment:
Ideally, we should be making assertions. Not, throwing custom errors. See previous tests for general boilerplate.

We want the console to show the assertion error with specifics like:
- `expected x, but received y`